src/Assets/Bugsnag/Editor/BugsnagEditor.cs

```diff
@@ -224,5 +253,135 @@ public static void OnPostProcessBuild(BuildTarget target, string path)
             File.WriteAllLines(pbxPath, lines.ToArray());
         }
 #endif
+
+        public static void EnableEDM()
+        {
+            try
+            {
+                EditDepsFile(true);
+                UpdateKotlinLibraryImportSettings(false);
+                AssetDatabase.Refresh();
```

Comment:
nit: I would move this Refresh call to `UpdateKotlinLibraryImportSettings` as presumably it is needed due to the library file change, which the function is responsible for.

Reply:
it's also needed for the deps file creation